docs/lua-modding/blueprints-with-lua.mdx

```diff
@@ -0,0 +1,135 @@
+# Using Blueprints with Lua
+When writing a Lua mod, you can call a method inside a ModActor blueprint. This is useful, as your Lua mod can hook into functions that aren't exposed via Blueprints, then you can create most of your mod's functionality within Blueprints. If you haven't yet, follow the steps to make a basic Blueprint mod in the [Creating A Blueprint Mod](creating-blueprint-mods/mod-setup-1.mdx) section, and then come back to this section.
+
+First, we have to let our Lua mod know that the Blueprints mod is active. We'll do this by creating a `Lua_ModInitialized` function in our blueprint like so:
+
+![1](./assets/blueprints-with-lua/1.png)
+
+This function will return your ModActor as a parameter to your Lua mod. Your Lua mod will then use your ModActor to call the functions inside of it. After making this `Lua_ModInitialized` function, we'll need to set an output parameter which will include our ModActor blueprint. This is what gets sent to our Lua mod so we can cache it and call our other functions found inside the blueprint.
+
+Your function should look like this:
+
+![1](./assets/blueprints-with-lua/2.png)
+
+:::note
+If you have multiple ModActor blueprints in the dropdown (which will happen if you've created more than one Blueprints mod), make sure the ModActor Object Reference is targeting your specific mod. You can tell if you're targeting your mod if you only see "ModActor" when hovering over the option, like so:
+
+![3](./assets/blueprints-with-lua/3.png)
+:::
```

Comment:
This can be avoided by creating another Actor blueprint within your mod folder and letting it handle all the logic instead. With this, ModActor is only responsible for spawning the 'core' Actor and you'll have easier time since you can just reference that 'core' Actor instead. I'll leave examples below.

ModActor spawning the Actor responsible for any logic:
![image](https://github.com/PalworldModding/Docs/assets/111317036/b91e8b34-a026-4eb6-82d8-78f70135339a)

Actor handling the actual logic:
![image](https://github.com/PalworldModding/Docs/assets/111317036/62211ef6-8091-40a8-811e-e40453f5ed20)

Folder:
![image](https://github.com/PalworldModding/Docs/assets/111317036/1e286751-2571-4f17-9be5-c61240884ac7)

Reply:
I used your phrasing and images to add a details section similar to some other sections in the documentation. How does this look?
![image](https://github.com/PalworldModding/Docs/assets/5509720/752b9dce-63cf-4c78-8157-9b6567b32f2a)

Reply:
This is good. Feel free to push the changes.

Reply:
Done, committed. Thanks!